components/brave_rewards/browser/test/common/rewards_browsertest_response.cc

```diff
@@ -246,9 +254,9 @@ void RewardsBrowserTestResponse::Get(
     if (url.find("credentials") != std::string::npos) {
       if (method == 0) {
         #if defined(OFFICIAL_BUILD)
-          *response = creds_tokens_prod_;
+          *response = creds_tokens_sku_prod_;
```

Comment:
we can not reuse signed creds from promotion flow for SKU flow, that's why we need to create new ones